recipes-extended/plugindeactivator/files/deactivate_plugin.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
```

Comment:
Please can this file get an Apache header (presumably Comcast)?
Not blocking though - will clear the check.